### PR TITLE
Add Indexa capital

### PR DESCRIPTION
**Pull request by frandieguez**

### CODE REVIEW

**Comment by phallobst** [comment]
Please rebase your branch to the current master.

**Comment by frandieguez** [comment]
rebased to master

**Comment by Carlgo11** [comment]
@frandieguez Are you going to rebase? Let me know otherwise and I'll close this pull request.

**Comment by frandieguez** [comment]
I have just rebased against master. Please let me know if this is what you expected

**Comment by Carlgo11** [comment]
Try:
```PWSH
git remote add upstream https://github.com/2factorauth/twofactorauth.git
git fetch upstream
git reset --hard upstream/master
git cherry-pick 4b3a518
git cherry-pick 5041d5b
git push origin master -f
```
When it's done there should only be 2 commits listed in this PR.
Let me know if you need further help.

**Comment by frandieguez** [comment]
Done, now only two commits are in the PR, sorry for the back-and-forth

File: _data/investing.yml [hunk 1]
```diff
@@ -108,6 +108,13 @@ websites:
     twitter: HealthEquity
     img: healthequity.png
 
+ - name: Indexa Capital
+    url: https://indexacapital.com/
+    img: https://secure.gravatar.com/blavatar/d2c2840e311c6596cdda5d8ae1b4f421?s=120
+    tfa:
+      - sms
+    doc: https://indexacapital.com/es/u/user#settings-login-activity
+
```
Comment (Carlgo11) [inline _data/investing.yml:117]:
```suggestion
    regions:
      - es

```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Alright now I can see your actual changes! 🥳 

Just a few things that need to be changed before we can merge:
* The image must be a file stored in `img/investing/`. I recommend you use this one:  and save it as indexa.svg.
* The documentation can't be reached without logging in. Either upload a screenshot of the doucmentation or screenshot(s) of the 2fa process. (I'll post a comment on how to upload screenshots below)
* Is the company only operating in Spain? If so, commit the suggested change's below.

**Comment by Carlgo11** [comment]
Screenshot note:
> Thanks for the pull request!
> 
> I see that you haven't add any documentation link. Could you maybe find one that describes how to enable 2FA on the site?
> 
> If no documentation is available or you can't find it, could you please add some screenshots of the 2FA setup process and/or login process?
> __Remember to blur or crop out any personal information linked to you.__
> 
> You can attach the screenshots here in a comment, like this:
> screenshot

**Comment by Carlgo11** [comment]
Oh no 😞 
Apparently indexacapital.com doesn't meet our requirement of being ranked <200K on Alexa.
It is currently above 1M (https://www.alexa.com/siteinfo/indexacapital.com)

Sadly this means the site can't be listed. Sorry for the inconvenience!

**Comment by frandieguez** [comment]
Oh! I understand sorry for the inconvenience

**Pull request closed without merging**